src/test/java/move/GeneralMovementTest.java

```diff
@@ -0,0 +1,171 @@
+package move;
+
+import static org.assertj.core.api.Assertions.*;
+
+import direction.Point;
+import java.util.List;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import piece.Piece;
+import piece.Pieces;
+import team.Team;
+
+public class GeneralMovementTest {
+
+    List<Piece> greenPieces = List.of(
+            new Piece("c", new Point(1, 10), new ChariotMovement()),
+            new Piece("c", new Point(9, 10), new ChariotMovement()),
+            new Piece("e", new Point(2, 10), new ElephantMovement(Team.GREEN.direction())),
+            new Piece("e", new Point(7, 10), new ElephantMovement(Team.GREEN.direction())),
+            new Piece("h",new Point(3, 10), new HorseMovement(Team.GREEN.direction())),
+            new Piece("h",new Point(8, 10), new HorseMovement(Team.GREEN.direction())),
+            new Piece("r",new Point(4, 10), new GuardMovement()),
+            new Piece("r",new Point(6, 10), new GuardMovement()),
+            new Piece("g",new Point(5, 9), new GeneralMovement()),
+            new Piece("n", new Point(2, 8), new CannonMovement()),
+            new Piece("n", new Point(8, 8), new CannonMovement()),
+            new Piece("s", new Point(1, 7), new SoldierMovement(Team.GREEN.direction())),
+            new Piece("s", new Point(3, 7), new SoldierMovement(Team.GREEN.direction())),
+            new Piece("s", new Point(5, 7), new SoldierMovement(Team.GREEN.direction())),
+            new Piece("s", new Point(7, 7), new SoldierMovement(Team.GREEN.direction())),
+            new Piece("s", new Point(9, 7), new SoldierMovement(Team.GREEN.direction()))
+    );
+
+    Pieces pieces = new Pieces(greenPieces);
+
+    @Test
+    @DisplayName("궁성 내 기물은 위로 한 칸 이동할 수 있다.")
+    void test1() {
+        // given
+        int x = 0;
+        int y = 0;
+        Point from = new Point(x, y);
+        Point to = new Point(x, y-1);
```

Comment:
```suggestion
        Point to = new Point(x, y - 1);
```
계속 같은 실수가 반복해서 보이네요.
커밋 전에 Lint로 코드 정리하시기 바랍니다.
intellJ에서 control + command + L 누르면 될거에요.

Reply:
커밋: [7039609](https://github.com/woowacourse/java-janggi/pull/72/commits/70396098124be41b32df4d01dadac194e450563e)

세심하게 봐야하는데 쉽지 않네요. 다음엔 주의하겠습니다. 감사합니다!